guide_to_notebooks.ipynb

```diff
@@ -0,0 +1,78 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2761f194-012c-4a1d-947c-e06ca0827d37",
+   "metadata": {},
+   "source": [
+    "# Cal-Adapt: Analytics Engine Navigation Guide"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a1a0d94d-d313-42cf-b45a-0dd31953daf8",
+   "metadata": {},
+   "source": [
+    "Welcome to the Cal-Adapt: Analytics Engine JupyterHub! Here, you can access tools and guidance to explore and analyze climate projections data downscaled for California. \n",
+    "\n",
+    "If you are new to working with Jupyter notebooks, check out our [video tutorial](https://www.youtube.com/watch?v=eEo9V_OJw9w) or review Jupyter's [introductory documentation](https://jupyter.org/try-jupyter/retro/notebooks/?path=notebooks/Intro.ipynb). You can also review our [technical documentation](https://climakitae.readthedocs.io/en/latest/) to learn more about the open source python library that powers these notebooks, `climakitae`.\n",
```

Comment:
This is so nitpicky-- sorry in advance-- but the Jupyter documentation capitalizes the "Notebook" in "Jupyter Notebook", so would be good to follow those conventions here as well.

Reply:
No problem, easy fix!